src/Quantum.Development.Kit/Microsoft.Quantum.Development.Kit.nuspec

```diff
@@ -20,7 +20,7 @@
     <tags>Quantum Q# QSharp</tags>
 
     <dependencies>
-      <dependency id="Microsoft.Quantum.Simulators" version="$version$" />
```

Comment:
Out of curiosity, what motivated hardcoding the version number here?

Reply:
the updateQDKVersion script updates it, so every time we pick a build tag we need to manually update it. 
There is no much value is using $version$ since the build does the right thing, so less work for us.